static/app/views/dashboards/widgetBuilder/components/filtersBar.tsx

```diff
@@ -15,35 +13,22 @@ function WidgetBuilderFilterBar() {
   const organization = useOrganization();
   const {selection} = usePageFilters();
   return (
-    <PageFiltersContainer
-      skipLoadLastUsed
-      disablePersistence
-      defaultSelection={{
-        datetime: {
-          start: null,
-          end: null,
-          utc: false,
-          period: DEFAULT_STATS_PERIOD,
-        },
-      }}
+    <Tooltip
+      title={t('Changes to these filters can only be made at the dashboard level')}
     >
-      <Tooltip
-        title={t('Changes to these filters can only be made at the dashboard level')}
-      >
-        <PageFilterBar>
-          <ProjectPageFilter disabled onChange={() => {}} />
-          <EnvironmentPageFilter disabled onChange={() => {}} />
-          <DatePageFilter disabled onChange={() => {}} />
-          <ReleasesProvider organization={organization} selection={selection}>
-            <ReleasesSelectControl
-              isDisabled
-              handleChangeFilter={() => {}}
-              selectedReleases={[]}
-            />
-          </ReleasesProvider>
-        </PageFilterBar>
-      </Tooltip>
-    </PageFiltersContainer>
+      <PageFilterBar>
+        <ProjectPageFilter disabled onChange={() => {}} />
+        <EnvironmentPageFilter disabled onChange={() => {}} />
+        <DatePageFilter disabled onChange={() => {}} />
+        <ReleasesProvider organization={organization} selection={selection}>
+          <ReleasesSelectControl
+            isDisabled
+            handleChangeFilter={() => {}}
+            selectedReleases={[]}
```

Comment:
Just noticed we have a bug here where we're not showing the selected releases properly. I'll fix that in another PR